src/components/billing/InvoiceForm.jsx

```diff
@@ -1,10 +1,6 @@
 import React, {Component} from "react";
 
 export default class InvoiceForm extends Component {
-    constructor(props) {
-        super(props);
-    }
-
```

Comment:
does this code serve no purpose at all?

Reply:
Nope, in reactjs if you put a constructor like that is just like if you did nothing, react shows a warning saying that you should remove it since it's the default constructor implementation.